cmd.go

```diff
@@ -259,11 +259,23 @@ type Info struct {
 	// For example, if this value is 'option', the default message 'value for flag'
 	// will become 'value for option'.
 	FlagKnownAs string
+
+	// ShowSuperFlags contains the names of the 'super' command flags
+	// that are desired to be shown in the sub-command help output.
+	ShowSuperFlags []string
 }
 
 // Help renders i's content, along with documentation for any
 // flags defined in f. It calls f.SetOutput(ioutil.Discard).
 func (i *Info) Help(f *gnuflag.FlagSet) []byte {
+	return i.HelpWithSuperFlags(nil, f)
+}
+
+// HelpWithSuperFlags renders i's content, along with documentation for any
+// flags defined in both command and its super command flag sets.
+// Only super command flags defined in i.ShowSuperFlags are displayed, if found.
+// It calls FLagSet.SetOutput(ioutil.Discard).
```

Comment:
I think you mean that it leaves the flagset in a state where it won't output anything, is that right? It would be clearer to say that in a higher-level way - this just repeats the code.

Reply:
I'll just remove it from the call I've added. cut and paste strikes again :D